card-scanning/src/main/java/com/adyen/checkout/card/scanning/AdyenCardScanner.kt

```
@@ -104,14 +103,16 @@ class AdyenCardScanner {
         }
     }
 
-    private fun startScanner(startIntentSender: (PendingIntent) -> Unit) {
+    private fun startScanner(startIntentSender: (PendingIntent) -> Unit): Boolean {
         val paymentCardRecognitionPendingIntent =
             paymentCardRecognitionPendingIntent ?: error("The scanner must be initialized before it can be started")
 
-        try {
+        return try {
             startIntentSender(paymentCardRecognitionPendingIntent)
+            true
         } catch (e: IntentSender.SendIntentException) {
-            throw CheckoutException("Failed to start payment card recognition", e)
+            adyenLog(AdyenLogLevel.ERROR, e) { "Failed to start payment card recognition" }
```

Do we not want to throw CheckoutException anymore?

---

Before the exception wasn't handled, so it would crash. If we propagate the exception to the merchant, then in drop-in we would show a dialog and after terminate the drop-in. That's a bit extreme for this case. So I figured ignoring the exception is best for now and we can improve in v6